scipy/stats/setup.py

```diff
@@ -27,9 +32,17 @@ def configuration(parent_package='',top_path=None):
     config.add_extension('mvn',
         sources=['mvn.pyf','mvndst.f'],
     )
-        
-    # What do I add here to build Rectangle?
-    
+
+    # Build cython rectangle demo
+    config.add_data_files('rect.pxd')
+    ext = config.add_extension(
+        'rect',
+        sources=['rect.cxx'],
```

Comment:
Scipy build system greps for PYX files and runs cython on them as one of the first steps of the build.  The CXX source files are the output that get compiled

Reply:
Got it.